changes-set.txt

```diff
@@ -26,6 +26,9 @@ make a github issue.)
 
 DONE:
 Date      Old       New         Notes
+22-Sep-23 spimevw   ---         duplicate
```

Comment:
This is not needed here (changes in the own mathbox need not be documented in the history).